### ISSUE TITLE
Add global directories directly to time machine skip list

### ISSUE THREAD

**Issue by jgoldhammer** [tg44/heptapod#7]
Thanks for your great tool. Works like a charm in macOS Monterey!

I want to exclude directories like ~/.yarn-cache/ or ~/.npm/  directly from my time machine backup.
These are 2 example for folders which I want to ignore completely in backups because they make the backup slow.

I looked into rules, but it seems to error prone to do it or too verbose at the moment.
Do you want to extend the rules syntax using the searchPaths for these folders and allow to exclude these folders with a simple hint?

```
name: "global directories"
enabled: true
searchPaths: ["~/.yarn-cache/", "~/.npm/"]
ignorePaths: []
ruleType: "ignore-directory"
```

You could disable the rule by default so that the user must activate it first.
What do you think?

**Comment by GBrachetta** [comment]
I believe you can already do that by running:

`heptapod rules ignoreAdd ~/.npm`

(or any other folder)

I agree this tool is amazing. It must get a lot more attention.

**Comment by tg44** [comment]
I'm thinking on a new type (not rule, something similar but above the rules, I'm still looking for the name), where every entry has the following fields;
 - name
 - paths: [] 
 - mode: followRules | globalIgnore | exclude
   - looking for better names :)
   - the idea is that we mostly want to exclude/ignore these

This would handle `.npm`, `.yarn`, etc. global cache/package folders.

**Comment by tg44** [comment]
I think this is implemented in #c3c21bd10 so I will close this.
